packcheck.sh

```diff
@@ -1764,6 +1780,7 @@ case $1 in
   cabal-new) die "cabal-new is not supported, please use cabal-v2 instead";;
   cabal-v2) shift; eval_env "$@"; BUILD=cabal-v2;;
   stack) shift; eval_env "$@"; BUILD=stack;;
+  hlint) shift; eval_env "$@"; BUILD=hlint;;
```

Comment:
Did you check all places where "$BUILD" is being used? At many places if it is not cabal build we assume stack build in the else case, now that we have hlint build we should consider that case too.

Reply:
Yes, those cases don't occur because of the final if-else statement. Maybe we could set "$COMMAND" and derive "$BUILD" accordingly.